### PR TITLE
[test] Run the karma tests in browserstack

### PR DESCRIPTION
**Pull request by oliviertassinari**
Cross-browser compatibility. We were missing the BrowserStack credentials in CircleCI.

Capture d’écran 2020-09-26 à 12 27 59

### CODE REVIEW

**Comment by oliviertassinari** [comment]
@dtassone The tests are run in:

Capture d’écran 2020-09-20 à 19 58 23

However, we have a couple of issues with these targets (e.g. `Object.entries` not defined). The grid isn't compatible. I think that it's important we configure browser to run the test in minimum browser versions we want to support. So, this time, we can't escape it: we decided to not support IE here (at least try and see the feedback), but what about the rest?

**What's the minimum browsers' version we aim to support?** :). It resonates with https://github.com/mui-org/material-ui/issues/15496 from @eps1lon.

File: netlify.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# [build]
+# # Directory (relative to root of your repo) that contains the deploy-ready
+# # HTML files and assets generated by the build. If a base directory has
+# # been specified, include it in the publish directory path.
+# publish = "docs/export/"
+
+# # Default build command.
+# command = "yarn docs:build && yarn docs:export && yarn storybook:build && yarn storybook:export && yarn demo:build && yarn demo:export"
+
+[build.environment]
```
Comment (oliviertassinari) [inline netlify.toml:10]: restored so we keep the version definition (help make sure it works in the lowest target we support)

File: .browserslistrc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+edge >= 83
+firefox >= 77
+chrome >= 83
+safari >= 11
+node 10.0
```
Comment (oliviertassinari) [inline .browserslistrc:5]: It looks like another advantage of not using TypeScipt to build the source but Babel. They don't seem to support browser target https://github.com/microsoft/TypeScript/issues/19183.

**Comment by eps1lon** [comment]
> It resonates with mui-org/material-ui#15496 from @eps1lon.

I'm definitely going to start with this the next week. Might not be the final target before release but it'll set up the final bundle structure and show in what direction the final bundle size goes.

**Comment by oliviertassinari** [comment]
The list of browsers can be queried at https://www.browserstack.com/docs/automate/api-reference/selenium/browser#get-browser-list.

**Review by dtassone — APPROVED** [review]

**Pull request merged**